cvat/apps/organizations/serializers.py

```diff
@@ -127,3 +146,35 @@ class Meta:
         model = Membership
         fields = ['id', 'user', 'organization', 'is_active', 'joined_date', 'role']
         read_only_fields = ['user', 'organization', 'is_active', 'joined_date']
+
+class AcceptInvitationSerializer(RegisterSerializer):
+    def validate_username(self, username):
+        return username
+
+    def validate_email(self, email):
+        return email
```

Comment:
the same comment as above

Reply:
We need to override `validate_email` because `RegisterSerializer` expects it to be unique, but we already have create user with such email

Reply:
It's clear